### ISSUE TITLE
Can the generated HTML page and JS/CSS resources be output separately?

### ISSUE THREAD

**Issue by zpshowhan** [TongchengOpenSource/smart-doc#490]
Can the generated HTML page and JS/CSS resources be output separately?
能否把生成的html页面和js、css资源分开输出？

**Comment by shalousun** [comment]
Currently, JS and CSS are also output to separate files. Generating HTML is not recommended; it's better to push the generated documents to Torna for long-term consideration.
现在js和css也是输出到单独的文件的，其实生成html这种也不是很推荐，长远考虑还是推送到torna这种比较符合

**Comment by zpshowhan** [comment]
Yes, you're right. HTML is generally used more frequently in everyday tasks. Here, it seems that HtmlApiDocBuilder places JS, CSS, and HTML all under one directory. Could the next version support this feature to output static resources and pages separately?
嗯，你说的到没错，日常一般还是用html还是比较多的，在这里 HtmlApiDocBuilder  看好像js、css、html都放到了一层目录下，是否下个版本可以支持下这个功能呢，可以把静态资源和页面分开输出？

**Comment by shalousun** [comment]
> 嗯，你说的到没错，日常一般还是用html还是比较多的，在这里 HtmlApiDocBuilder 看好像js、css、html都放到了一层目录下，是否下个版本可以支持下这个功能呢，可以把静态资源和页面分开输出？

Do you want to separate the directories? What's the use case?
你们的是想分开目录放？使用场景是什么
